app/controllers/ballots_controller.rb

```diff
@@ -47,7 +52,7 @@ def load_ballot
 
   def load_all_beers
     @beers =
-      if brewery = Poll.current.brewery.presence
+      if brewery = Poll.current.try(:brewery)
```

Comment:
Presence returns the string, not a boolean. We're assigning the value to `brewery`.

Aaahhhh, the issue is that there might not be a current poll.